user-service/src/main/java/uk/gov/gchq/palisade/service/user/model/UserRequest.java

```diff
@@ -31,11 +31,10 @@
 import java.util.StringJoiner;
 
 /**
- * The UserRequest is the input for user-service to identify the User associated with the given User ID.
- * UserResponse is the output for this service which will include the User identified by the service.
+ * The UserRequest is the input for User Service to identify the User associated with the given User ID.
  * Note there are two classes that effectively represent the same data but represent a different stage of the process.
- * uk.gov.gchq.palisade.service.palisade.request.OriginalRequest is the client request that has come into the Palisade Service.
- * uk.gov.gchq.palisade.service.user.request.UserRequest is the input for the user-service.
+ * uk.gov.gchq.palisade.service.palisade.model.PalisadeSystemResponse is the output from the Palisade Service.
```

Comment:
reword output from, something like, PalisadeSystemResponse is sent from the Palisade Service after a valid request is accepted

Reply:
This has been re-worded.